src/librustc_mir/interpret/traits.rs

```diff
@@ -24,24 +24,32 @@ impl<'a, 'mir, 'tcx, M: Machine<'a, 'mir, 'tcx>> EvalContext<'a, 'mir, 'tcx, M>
     pub fn get_vtable(
         &mut self,
         ty: Ty<'tcx>,
-        trait_ref: ty::PolyTraitRef<'tcx>,
+        poly_trait_ref: Option<ty::PolyExistentialTraitRef<'tcx>>,
     ) -> EvalResult<'tcx, Pointer<M::PointerTag>> {
```

Comment:
Please add a comment here saying which other method this has to be kept in sync with, and add a matching comment on the other end.

Reply:
Oh never mind, you did.  That's the issue with reviewing commits individually.^^